posts/2019/concerts-this-fall.rst

```diff
@@ -7,18 +7,14 @@
 .. description: 
 .. type: text
 
-We are happy to announce further information on the upcoming concerts in
```

Comment:
All of the posts are already published.
This is basically rewriting history, which is ugly for RSS readers.

The way to work with this is to *review* and then publish them, not to rewrite them arbitrarily afterwards in drive-by-commits.